MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -45,6 +45,10 @@ internal sealed class MixedRealityExtensionApp : IMixedRealityExtensionApp, ICom
         private IList<GameObject> _ownedGameObjects = new List<GameObject>();
         private Queue<CreateFromGLTF> _createFromGLTFQueue = new Queue<CreateFromGLTF>();
 
+        private Dictionary<Guid, AudioSource> _soundInstances = new Dictionary<Guid, AudioSource>();
```

Comment:
You have some old audio fields. Bad merge? These moved to SoundManager.cs.

Reply:
Yeah.  They snuck in during my initial rebase to red.  Removed.